### ISSUE TITLE
Future of Helm Classic and Deis Workflow

### ISSUE THREAD

**Issue by miracle2k** [helm/helm-classic#492]
I was (unsuccessfully) searching for the story behind the split between Helm and Helm Classic. What is the plan for Helm Classic? Is it only maintained until the new Helm is ready? Is this a permanent split? Will Deis Workflow use the new Helm in the future?

**Comment by bacongobbler** [comment]
You're correct on all three points. Helm (the new Helm) is a joint project between Deis and Google. Helm "classic" is being lightly maintained (as in only bugfixes). It's not a split in the sense that kubernetes/helm is a fork of helm classic but rather the successor to helm classic. There are many limitations in regards to the initial approach's infrastructure (being all client-side, whereas k8s/helm is both server-side and client-side), so we had to go about it another way and EOL this project. Workflow will use Helm in the future, and in fact we are currently standing up charts in each repo for the Workflow components. See https://github.com/deis/charts/issues/350

Hope this helps!

**Comment by slack** [comment]
@bacongobbler beat me to the punch! See the youtube link below for a presentation from the Deis community meeting in May with more context...

We have placed Helm Classic in maintenance mode and will shut down the project when:
* Kubernetes Helm reaches a stable release: soon!
* Workflow charts have been updated for new Helm: progress [here](https://github.com/deis/charts/issues/350) and [here](https://github.com/deis/charts/issues/349)
* We have a reasonable upgrade process for Workflow users between their Helm Classic installs and Kubernetes Helm

If you would like a bit of background on the Helm Classic to Kubernetes Helm story check out <https://youtu.be/Jb-X_yVE2-w?t=7m55s>, for a bit more color on the design differences and goals.
